contact/tests/test_forms.py

```diff
@@ -8,6 +8,7 @@ class TestSignupForm:
     """
     Test class for the Enquiry form and it's methods
     """
+
     from contact import forms, models
```

Comment:
How come imports are within the class?

Reply:
This is one of the issues I was having. It's more or less the models import that caused the issue in a couple locations as it means that when pytest reaches that file, it'll try to import and throw an error because there isn't a django settings module. Adding the imports inside the class and marking them as "skip if you can't import django.settings" is the cleanest way around it (at least that I found)